tests/testthat/test-install-spark.R

```diff
@@ -4,7 +4,7 @@ test_that("supported spark_versions can be downloaded", {
   skip_on_cran()
   test_requires("RCurl")
 
-  versions <- spark_versions(latest = FALSE)
+  versions <- sparkinstall:::spark_versions(latest = FALSE)
```

Comment:
Does this function need to be exported? (We shouldn't use `:::` when referring to functions from other packages)